src/std/sugar.ss

```diff
@@ -25,6 +25,10 @@
   chain
   is)
 
+(import
+  (for-syntax :gerbil/expander)
```

Comment:
can we narrow this import so that we can add the necessary identifiers to the prelude expande module in a subsequent pr?

Reply:
I believe it was for `core-bound-identifier?`. Is that an identifier you want to include in the prelude expander module?

Reply:
yeah, we should consider it as it is generally useful. 
We can do in a subsequent pr, no need to do it here.